src/Azure/AzureAD/Authentication.AzureAD.UI/src/AzureADJwtBearerOptionsConfiguration.cs

```diff
@@ -30,8 +30,10 @@ public void Configure(string name, JwtBearerOptions options)
                 return;
             }
 
-            options.Audience = azureADOptions.ClientId;
-            options.Authority = new Uri(new Uri(azureADOptions.Instance), azureADOptions.TenantId).ToString();
+            options.Audience = string.Format(azureADOptions.Audience?.Replace("{ClientId}", "{0}"),
+                                             azureADOptions.ClientId);
+            options.Authority = string.Format(azureADOptions.Authority.Replace("{Instance}", "{0}").Replace("{TenantId}", "{1}"),
+                                              azureADOptions.Instance, azureADOptions.TenantId);
         }
 
         public void Configure(JwtBearerOptions options)
```

Comment:
This method doesn't seem to do anything.

Reply:
This is fine, as this class only targets named configuration instances.

Reply:
A code comment might be useful, then.